README.md

```diff
@@ -1 +1,129 @@
-# scalardb-samples
\ No newline at end of file
+# Scalar DB server Samples
+- This repository contains the sample application that will connect to `Scalar DB server` as backend. For using native `Scalar DB` library to connect to `Scalar DB` directly please refer to this document [`Getting Started`](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started.md).
+- `Scalar DB server` is a `gRPC` server that implements Scalar DB interface. More information about `Scalar DB server` can be found [`here`](https://github.com/scalar-labs/scalardb/tree/master/docs/scalardb-server.md)
+
+## Our sample application
+A simple electronic money cli app which has the following features:
+- Charge an `amount` to a `user_id`
+- Pay an `amount` from a `user_id` to another `user_id`
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Setting up
+### Scalar DB server
+- In this sample we will use Cassandra as storage for our Scalar DB server. Cassandra will also run along with Scalar DB server in containers. The setting for our Scalar DB server is [`database.properties`](./database.properties)
+```
+# Comma separated contact points
+scalar.db.contact_points=cassandra
+
+# Port number for all the contact points. Default port number for each database is used if empty.
+scalar.db.contact_port=9042
+
+# Credential information to access the database
+scalar.db.username=cassandra
+scalar.db.password=cassandra
+
+# Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set. Default storage is cassandra.
+scalar.db.storage=cassandra
+```
+- To start our Scalar DB server, we use the below command (please note that we should wait around a bit more than one minute to make sure every container is already up, because Scalar DB container has to wait for Cassandra container to be fully started)
+```
+docker-compose -f docker-compose-cassandra.yml up -d
+```
+- *For using other databases as backend for Scalar DB server we can change the configuration in [`database.properties`](database.properties) according to [`Getting Started`](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started.md). After that we can start our `Scalar DB server` using this [`docker-compose.yml`](docker-compose.yml) instead.*
+### Scalar DB Client
+- Our sample application will connect to `Scalar DB server` via gRPC, the configuration basically has the same format as our server configuration, the difference is we have to change storage to gRPC and transaction manager to gRPC as well. The configuration is [`scalardb-client.properties`](scalardb-client.properties).
```

Comment:
I think this part is pretty tricky since the client used in the sample implements Scalar DB interface as well.

```suggestion
The sample application uses a client that implements Scalar DB interface. Thus, you can configure the client in the same way as the server-side. But, in this case, you need to specify the server as a contact point and `grpc` for the storage and transaction_manager configuration as follows. (it is stored in [scalardb-client.properties](scalardb-client.properties)).
```